environments/production/common/alb.tf

```diff
@@ -50,17 +50,17 @@ module "alb" {
       priority         = 17
     }
 
-    # backend_uk = {
-    #   paths            = ["/api/*", "/uk/api/*"]
-    #   healthcheck_path = "/healthcheckz"
-    #   priority         = 20
-    # }
-    #
-    # backend_xi = {
-    #   paths            = ["/xi/api/*"]
-    #   healthcheck_path = "/healthcheckz"
-    #   priority         = 21
-    # }
+    backend_uk = {
+      paths            = ["/new/api/*", "/new/uk/api/*"]
```

Comment:
I'm enabling the backend routes to a `/new` path to avoid an issue with production deployments which requires the target group we're creating to have a listener